tests/onnx_backend_test.py

```diff
@@ -12,5 +12,10 @@
 # import all test cases at global scope to make them visible to python.unittest
 globals().update(onnx.backend.test.BackendTest(c2.Caffe2Backend, __name__).test_cases)
 
+OnnxBackendNodeTest.test_rnn_gru_simple = unittest.skip(
```

Comment:
nit: maybe make it prettier and have some helper function doing it? E.g. `OnnxBackendTest.add_skip('test_rnn_gru_simple')`?